config/locales/ja.yml

```diff
@@ -226,6 +231,7 @@ ja:
       content: セキュリティ認証に失敗しました。Cookieをブロックしていませんか？
       title: セキュリティ認証に失敗
     '429': リクエストの制限に達しました。
+    noscript: Mastodonを利用する場合はJavaScriptを有効にしてください。
```

Comment:
'利用' is good but it might sound like saying about Mastodon as a service, not only our web client. So it may be better with '使用' '使う', or rephrase it to something like:

- JavaScript を有効にしてください
- このページを表示するには JavaScript が必要です

but this is not a huge problem I think.

(Mentioning something like "Instead, you can use Mastodon from 3rd-party apps" may also be good, but this will be no longer translation issue)